### ISSUE TITLE
1.0.0-rc.1: "npm insta"

### ISSUE THREAD

**Issue by boredland** [drone/drone#2523]
My pipeline looks like this:
```
kind: pipeline
name: default

steps:
- name: build
  image: node:8-alpine
  commands:
  - npm install
```
Drone logs the following:
```
+ npm insta

Usage: npm <command>

where <command> is one of:
...
npm@6.4.1 /usr/local/lib/node_modules/npm

Did you mean one of these?
    init
    install
    unstar
```
So clearly the "ll" went missing somewhere.

**Comment by boredland** [comment]
My first try to work around was to escape the line like that:
```
kind: pipeline
name: default

steps:
- name: build
  image: node:8-alpine
  commands:
  - "npm --verbose install"
```
That leads the project to not even start the "default" pipeline (thus not even showing stages or error output), not relevant debugging info either.

**Comment by bradrydzewski** [comment]
I have tried multiple times and cannot reproduce any issue. I tried with both examples. We can re-open this if core members can reproduce. Per our issue template, we politely ask that you use our mailing list for support and issue triage prior to creating GitHub issues (as we use this as a to-do list). Thanks.

> Bugs or Issues? Please do not open a GitHub issue until you have
discussed and verified with community support:  https://discourse.drone.io/
